src/PersistentSubscriptions/PersistentSubscriptionsClient.php

```diff
@@ -34,10 +34,11 @@ class PersistentSubscriptionsClient
     private HttpClient $client;
     private int $operationTimeout;
 
-    public function __construct(int $operationTimeout)
+    public function __construct(int $operationTimeout, bool $tlsTerminatedEndpoint = false)
```

Comment:
@prolic Hmm... what about using https by default?

Reply:
personally I think this should stay with http by default. will this not be mostly that case?

Reply:
we don't need a default value here, because it's only constructed via the manager.

Reply:
https is default for event-store v6, while http is default for v5. As long as v6 isn't in final release, let's stick to http for now.

Reply:
@prolic shouldn't client be passed as a dependency instead of making new instance in constructor?

Reply:
nope, that's only an implementation detail, not an inject-able dependency.